coriolis/utils.py

```diff
@@ -336,6 +336,9 @@ def check_fs(ssh, fs_type, dev_path):
         LOG.debug("File system checked:\n%s", out)
     except Exception as ex:
         LOG.warn("Checking file system returned an error:\n%s", str(ex))
```

Comment:
Please change the `str(ex)` to `utils.get_exception_details()` while you're here.